pages/GSoC.md

```diff
@@ -341,6 +341,17 @@ should be opened in DispVM or locally)
 
 **Mentor**:
 
+### New Qubes Manager
+**Project**: New Qubes Manager
```

Comment:
This have some progress already:
 - https://github.com/QubesOS/qubes-issues/issues/2132#issuecomment-255106656
 - https://github.com/kalkin/manager-new
 - https://github.com/kalkin/qubes-dbus